datalad_debian/new_package.py

```diff
@@ -23,7 +23,18 @@
 
 @build_doc
 class NewPackage(Interface):
-    """Create a new package dataset
+    """Create a new package dataset inside of a distribution dataset
+
+    In its final stage, a typical package dataset contains the source files,
+    built binaries, and builder for a Debian package.
+    This command creates the initial structure for this: A package dataset in
+    the 'package/' subdirectory of a distribution dataset, and a 'builder'
```

Comment:
Remove the `,`. It's a two item list, not two sentences.

Reply:
thx